packages/kbn-text-based-editor/src/esql_documentation_sections.tsx

```diff
@@ -1618,6 +1618,44 @@ NOTE: \`MV_DEDUPE\` may, but won’t always, sort the values in the field.
         />
       ),
     },
+    {
+      label: i18n.translate(
+        'textBasedEditor.query.textBasedLanguagesEditor.documentationESQL.mvFirstFunction',
+        {
+          defaultMessage: 'MV_FIRST',
+        }
+      ),
+    description: (
+      <Markdown
+        openLinksInNewTab={true}
+        markdown={i18n.translate(
+          'textBasedEditor.query.textBasedLanguagesEditor.documentationESQL.mvFirst.markdown',
+          {
+            defaultMessage: `### MV_FIRST
+Converts a multivalued field into a single valued field containing the first value. This is most useful when reading from a function that emits multivalued fields in a known order like \`SPLIT\`.
+
+For example:
+
+\`\`\`
+ROW a="foo;bar;baz" 
+| EVAL first_a = MV_FIRST(SPLIT(a, ";"))
+\`\`\`
```

Comment:
@dej611 We need to add this in our validator

Reply:
good catch! I've annotated them into ES and then forgot to add them here in Kibana 🤦

Reply:
Done => https://github.com/elastic/kibana/pull/175518

Reply:
A true gem ❤️